app/assets/javascripts/answers.js

```diff
@@ -35,8 +33,24 @@ function initUserFeedback() {
     var $button = $(ev.target).closest('.btn-feedback');
     var feedbackActive = $button.data('feedback-active');
     var feedbackString = $button.data('feedback-string');
-    var feedbackQid = $button.data('feedback-qid');
+    var feedbackQid = quizQuestions.questions[quizAllIndex].id;
```

Comment:
This function `initUserFeedback()` is also called from the [questions#show page](https://github.com/penzance/askup/blob/master/app/views/questions/show.html.erb#L61), and changing it this way breaks that function. So it would be best to find some way to call separate functions, e.g.
`initUserFeedbackQuizAll()` and `initUserFeedbackQuestionShow()`.